crates/pixi_config/src/lib.rs

```diff
@@ -1008,6 +1023,13 @@ impl Config {
         // Enable sharded repodata by default.
         config.repodata_config.default.disable_sharded = Some(false);
 
+        // HACK: Use win-64 as the default tool platform if currently running on
+        // win-arm64. This is a workaround for the fact that we don't have a
+        // good win-arm64 toolchain yet.
+        if Platform::current() == Platform::WinArm64 {
+            config.tool_platform = Some(Platform::Win64);
+        }
+
```

Comment:
I think we should trace warning in this case.

Reply:
This function is only called by tests where most of the time we dont have tracing setup.